src/User32.Desktop/User32+WINDOW_STATION_ACCESS_MASK.cs

```diff
@@ -0,0 +1,308 @@
+        /// /// This is a set of Specific Rights bitmask that defines access rights and security for desktop objects
+        /// </summary>
+        /// <remarks>
+        /// <see cref="Kernel32.ACCESS_MASK"/> is quite well described here: http://blogs.msdn.com/b/openspecification/archive/2010/04/01/about-the-access-mask-structure.aspx
+        /// Window Station security and access rights are quite well described here: https://msdn.microsoft.com/en-us/library/windows/desktop/ms687391(v=vs.85).aspx
+        /// </remarks>
+        public struct WINDOW_STATION_ACCESS_MASK : IComparable, IFormattable, IComparable<WINDOW_STATION_ACCESS_MASK>, IEquatable<WINDOW_STATION_ACCESS_MASK>, IComparable<Kernel32.ACCESS_MASK>, IEquatable<Kernel32.ACCESS_MASK>
+        {
+            /// <summary>
+            /// Bits 28-31
+            /// </summary>
+            private const uint GenericRightsMask = 0xf0000000;
+
+            /// <summary>
+            /// Bits 24-27
+            /// </summary>
+            private const uint SpecialRightsMask = 0x0f000000;
+
+            /// <summary>
+            /// Bits 16-23
+            /// </summary>
+            private const uint StandardRightsMask = 0x00ff0000;
+
+            /// <summary>
+            /// Bits 0-15
+            /// </summary>
+            private const uint SpecificRightsMask = 0x0000ffff;
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="WINDOW_STATION_ACCESS_MASK"/> struct.
+            /// </summary>
+            /// <param name="value">The value for the <see cref="WINDOW_STATION_ACCESS_MASK"/>.</param>
+            public WINDOW_STATION_ACCESS_MASK(uint value)
+            {
+                this.Value = value;
+            }
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="WINDOW_STATION_ACCESS_MASK"/> struct.
+            /// </summary>
+            /// <param name="value">The reference <see cref="Kernel32.ACCESS_MASK"/> for the <see cref="WINDOW_STATION_ACCESS_MASK"/>.</param>
+            public WINDOW_STATION_ACCESS_MASK(Kernel32.ACCESS_MASK value)
+            {
+                this.Value = value;
+            }
+
+            /// <summary>
+            /// Generic access rights for the interactive window station object, which is the window station assigned to the logon session of the interactive user.
+            /// </summary>
+            [Flags]
+            public enum InteractiveWindowStationGenericRight : uint
+            {
+                GENERIC_READ = Kernel32.ACCESS_MASK.StandardRight.STANDARD_RIGHTS_READ |
+                    SpecificRight.WINSTA_ENUMDESKTOPS |
+                    SpecificRight.WINSTA_ENUMERATE |
+                    SpecificRight.WINSTA_READATTRIBUTES |
+                    SpecificRight.WINSTA_READSCREEN,
+
+                GENERIC_WRITE = Kernel32.ACCESS_MASK.StandardRight.STANDARD_RIGHTS_WRITE |
+                    SpecificRight.WINSTA_ACCESSCLIPBOARD |
```

Comment:
Similar question as elsewhere: where is `GENERIC_WRITE` defined in terms of these constants in native code? Otherwise where did you get these definitions?